MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -152,7 +152,12 @@ public class NavigationDirections: Directions {
             let tilePath = filePathURL.path
             let outputPath = outputDirectoryURL.path
             
-            let numberOfTiles = Navigator().unpackTiles(forPackedTilesPath: tilePath, outputDirectory: outputPath)
+            let navigator: Navigator = {
+                let settingsProfile = SettingsProfile(application: ProfileApplication.kMobile,
+                                                      platform: ProfilePlatform.KIOS)
+                return Navigator(profile: settingsProfile, customConfig: "")
+            }()
```

Comment:
Can we use `self.navigator` here instead of initializing a new one?

Reply:
In this place navigator is used in static context, probably better to leave it as it is.